lib/id3tag/frames/v2/table_of_contents_frame.rb

```diff
@@ -0,0 +1,49 @@
+module ID3Tag
+  module Frames
+    module V2
+      class TableOfContentsFrame < BasicFrame
+        FLAG_A_TOP_LEVEL_BIT = "00000001".to_i(2)
+        FLAG_B_ORDERED_BIT = "00000010".to_i(2)
+
+        def element_id
+          @element_id ||= StringUtil.split_by_null_byte(raw_content).first
+        end
+
+        def flags
+          parts[:flags].unpack1("C")
+        end
+
+        def top_level?
+          flags & FLAG_A_TOP_LEVEL_BIT > 0
+        end
+
+        def ordered?
+          flags & FLAG_B_ORDERED_BIT > 0
+        end
+
+        def entry_count
+          @entry_count ||= parts[:entry_count].unpack1("C")
+        end
+
+        def child_element_ids
```

Comment:
Is this the main content of the frame? Might be good idea to define method alias `content` and `inspectable_content`

Reply:
👍 Done in 346e5abcd6894dd071abf83972e8affdd4f53ec8